src/assets/translations/en-us.yaml

```diff
@@ -4574,8 +4574,14 @@ troubleshooting:
     title: Kubernetes
     resetKubernetes:
       title: Reset Kubernetes
-      description: Resetting Kubernetes will delete workloads and configuration.
+      description:  Resetting Kubernetes will delete all workloads and configuration.
```

Comment:
The linter doesn't care, but please remove the extra space before "Resetting".

Reply:
Good catch, thanks!